src/linear-memory.js

```diff
@@ -0,0 +1,36 @@
+import { HeapVerifier } from "./heap-verifier";
+
+export class LinearMemory {
+  constructor({ initial = 2048, maximum = 2048 }) {
```

Comment:
As long as you are able to grow memory, I would suggest `initial = 256, maximum = 2048`